integration-tests/legend-docker-setup/standalone-engine/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+
+services:
+  engine:
+    container_name: engine
+    image: finos/legend-engine-server:snapshot
+    ports:
+      - 6060:6060
+    networks:
+      - legend
+    volumes:
+      - ./config:/config
+    # NOTE: the Java CLI wildcard must be wrapped by quotes
+    # See https://stackoverflow.com/questions/14722657/java-classpath-wildcard-behaviour/14722763
+    command: >
```

Comment:
Do we want to redefine the command the container launches with - or can we just rely on the command that's already present in the legend-engine-server Dockerfile?

Reply:
Yes. We should change that. Let's get this in first as a baseline.